cmd/cloud/deployment.go

```diff
@@ -214,6 +217,13 @@ func newDeploymentUpdateCmd(out io.Writer) *cobra.Command {
 	if organization.IsOrgHosted() {
 		cmd.Flags().StringVarP(&schedulerSize, "scheduler-size", "", "", "The size of Scheduler for the Deployment. Possible values can be small, medium, large")
 		cmd.Flags().StringVarP(&highAvailability, "high-availability", "a", "", "Enables High Availability for the Deployment")
+		cmd.Flags().StringVarP(&defaultTaskPodCPU, "default-task-pod-cpu", "", "", "The Default Task Pod CPU to use for the Deployment. Example value: 0.25")
```

Comment:
Similar to other comment - this number doesn't include units and might need them if we add them to the other one.

Reply:
CORE API does not want units for CPU